packages/xdl/src/project/Doctor.js

```diff
@@ -472,9 +480,17 @@ export async function getExpoSdkStatus(projectRoot: string): Promise<ExpoSdkStat
 
   try {
     let sdkPkg;
-    if (pkg.dependencies['exponent']) {
+    if (pkg.dependencies && pkg.dependencies['exponent']) {
+      sdkPkg = 'exponent';
+    } else if (pkg.dependencies && pkg.dependencies['expo']) {
+      sdkPkg = 'expo';
+    } else if (pkg.devDependencies && pkg.devDependencies['exponent']) {
+      sdkPkg = 'exponent';
+    } else if (pkg.devDependencies && pkg.devDependencies['expo']) {
+      sdkPkg = 'expo';
+    } else if (pkg.peerDependencies && pkg.peerDependencies['exponent']) {
```

Comment:
No need to make this code support the legacy `exponent` dependency. None of the currently supported SDK versions were published as `exponent` anymore.

Reply:
I feel like that would be a breaking change needing a major semver bump, someone may have mapped exponent manually